frontend/src/metabase/setup/actions.ts

```diff
@@ -44,6 +44,9 @@ export const goToNextStep = createAsyncThunk(
     const state = getState() as State;
     const nextStep = getNextStep(state);
     dispatch(selectStep(nextStep));
+    if (nextStep === "completed") {
```

Comment:
`setEmbeddingHomepageFlags` was called on the last step, but the last step is now dynamic based on if we have the analytics opt out or not.
This makes sure we're calling `setEmbeddingHomepageFlags` even if we change the steps again in the future